.travis.yml

```diff
@@ -14,6 +14,8 @@ install:
 
 script:
  - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.10 --no-upstream-fetch && python3 makeBuild.py
+ - python3 build.py --distro openshift-dedicated --product "OpenShift Dedicated" --version 4 --no-upstream-fetch && python3 makeBuild.py
+ - python3 build.py --distro openshift-rosa --product "Red Hat OpenShift Service on AWS" --version 4 --no-upstream-fetch && python3 makeBuild.py
 
 after_success:
```

Comment:
We might also need to determine what constitutes a success if multiple distro builds are being run. Would one successful build cause the automerge to take place, or would all of the `build.py` runs need to succeed? How can we test this in Travis?